lib/gooddata/lcm/user_bricks_helper.rb

```diff
@@ -0,0 +1,32 @@
+module GoodData
+  module LCM2
+    # Contains code shared by Users Brick and User Filters Brick
+    class UserBricksHelper
+      class << self
+        def resolve_client_id(domain, project, data_product)
+          md = project.metadata
```

Comment:
Refers to 'project' more than self (maybe move it to another class?) (FeatureEnvy)